aboutstyles.css

```diff
@@ -0,0 +1,397 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Lato", sans-serif;
+  overflow-x: hidden;
+}
+
+.nav-menu-about {
+  background-color: #fff;
+  background-image: url('./images/topbg.png');
+  background-size: cover;
+  height: 120px;
+}
+
+.mobile-menu {
+  min-height: 100vh;
+  width: 100vw;
+  display: none;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  z-index: 20;
+  background: #e926102f;
+  backdrop-filter: blur(10px);
+  transition: 0.6s;
+}
+
+.mobile-menu ul {
+  list-style-type: none;
+  margin: 10% 5%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.mobile-menu ul li {
+  padding: 10% 5%;
+}
+
+.mobile-menu li a {
+  text-decoration: none;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+#cross {
+  height: 60px;
+  width: 75px;
+}
+
+.imageone-about {
+  width: 60px;
+  height: 60px;
+  cursor: pointer;
+}
+
+.nav-menu-desktop-about {
+  display: none;
+}
+
+.list2-about {
+  display: none;
+}
+
+.nav-menu-desktop2-about {
+  display: none;
+}
+
+.headline-about {
+  background-image: url('./images/bgimagedesign.png');
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: 250px 0;
+  background-color: #d3d3d3;
+  padding: 45px;
+  width: 100%;
+  height: 700px;
+}
+
+.topheadingh3-about {
+  color: #ec5242;
+}
+
+.topheadingh1-about {
+  font-size: 47px;
+  display: block;
+  color: red;
+  font-weight: 900;
+  background-image: linear-gradient(to bottom right, red, yellow);
+  background-size: 100%;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: #ec0e0e4f;
+}
+
+.paradiv-about {
+  background-color: #fff;
+  border: 1px solid darkgrey;
+  padding: 10px;
+  width: 100%;
+}
+
+.headlinepara-about {
+  color: #272a31;
+}
+
+.datediv-about {
+  width: 100%;
+}
+
+.dateheading-about {
+  color: #272a31;
+  font-weight: 600;
+}
+
+.locationpara-about {
+  color: #272a31;
+}
+
+.headlinedesktop-about {
+  display: none;
+}
+
+.hr {
+  width: 30px;
+  border-color: #ec5242;
+  margin-bottom: 16px;
+}
+
+.logo-section {
+  width: 100%;
+  height: 550px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
```

Comment:
- [ ] Kindly make sure to use the same background color on this logo section in order to match the provided design as much as possible.

  - ### YOUR DESIGN
  
     ![image](https://user-images.githubusercontent.com/87197412/219406875-d9e84a43-f1d6-4153-b0ed-db757ecf569e.png)

  - ### PROVIDED DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/219407033-bac43b69-c741-4205-8e07-fa39a95f2b8f.png)